tests/jballerina-unit-test/src/test/resources/test-src/bala/test_projects/tuple_type_project/functions_returning_tuples.bal

```diff
@@ -12,6 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+public const annotation member on field;
+public const annotation tup on type;
+
+type T1 [int, @member int, string...];
+type T2 [int, @member int, string];
+type T3 [@member int, string];
+
+@tup
+type T4 [@member int, string];
+
+public function returnTupleAnnot1() returns T1 {
```

Comment:
Let's revisit the names. This returns tuple whose member type descriptors have annotations. The current name kinda implies that it is returning annotations of a tuple.